google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkService.java

```diff
@@ -27,6 +27,9 @@ static CloudSdkService getInstance() {
     return ServiceManager.getService(CloudSdkService.class);
   }
 
+  /** Called when this service becomes primary choice for serving Cloud SDK. */
+  void activate();
```

Comment:
nit: how about setActive instead of activate?

Reply:
I think this is good, except maybe seeing `setActive` could imply we want to pass a property there, like `setActive(true|false)`, could be a bit misleading?

Reply:
yeah thats true. I'm ok leaving it.